OWML.Common/IModConsole.cs

```diff
@@ -1,8 +1,17 @@
-﻿namespace OWML.Common
+﻿using System;
+
+namespace OWML.Common
 {
     public interface IModConsole
     {
+        [Obsolete]
         void WriteLine(string s);
+        [Obsolete]
         void WriteLine(params object[] s);
+
+        void WriteLine(string sender, MessageType type, string s);
```

Comment:
Mods and OWML have their own console/logging objects initialized with manifests, to avoid giving sender explicitly when logging.
It doesn't feel right that when i log in my mod I can lie about who I am this easily.

Reply:
In other words: remove sender parameter, take it from manifest instead

Reply:
done